### PR TITLE
V1.1 add target config

### PR DESCRIPTION
**Pull request by Namsulee**
Update target config for tizen-minimal according to target.
target-config.py includes proxy information.
If somebody wants to set proxy in each target, do 'python target-config.py'
The service file of dockze-launcher will be edited and then proxy will be applied if you reboot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,85 @@
...
-		$ make artik710_defconfig
-		$ make
+## Tizen for Host OS
+### RaspberryPI 3 
+You have to prepare,
+- prerequisite (SD card)
+```
+$ sudo apt-get install pv
+$ chmod u+x sd_fusing_rpi3_docker.sh
+$ sudo ./sd_fusing_rpi3_docker.sh -d /dev/sdb --format
```
Comment (sy1124kim) [inline README.md]: You should be change /dev/sdb to /dev/sdX.

File: README.md [hunk 2]
```diff
@@ -1,63 +1,85 @@
...
+```
+- flash image with dockzen F/W into SD card
+```
+$ cd ./minimal/raspberrypi3/tizen-minimal
+$ sudo ./sd_fusing_rpi3_docker.sh -d /dev/sdb -b tizen-boot_xxxxxxxx.tar.gz
```
Comment (sy1124kim) [inline README.md]: You should be change /dev/sdb to /dev/sdX.

File: README.md [hunk 3]
```diff
@@ -1,63 +1,85 @@
...
+$ sudo ./sd_fusing_rpi3_docker.sh -d /dev/sdb -b tizen-minimal_xxxxxxxx.tar.gz
```
Comment (sy1124kim) [inline README.md]: You should be change /dev/sdb to /dev/sdX.

**Comment by sy1124kim** [comment]
You should be change /dev/sdb to /dev/sdX.

**Comment by jiehwan** [comment]
replace F/W into FW

**Comment by Namsulee** [comment]
Fixed that you guys requested

**Pull request merged**